README.md

```diff
@@ -12,6 +12,12 @@ This style guide is a compilation of [atomic components](http://bradfrost.com/bl
 - Link to the production files at `styleguide/public/assets/`
 
 ## To begin working:
+**Environment setup (mac)**
+
+ - Have [`homebrew`](https://brew.sh/) installed
+ - `brew install nvm`
```

Comment:
Strange... i put:
``` export NVM_DIR="$HOME/.nvm"
      . "#{opt_prefix}/nvm.sh"```
into my `.profile` and it didn't work for me... maybe i just messed something up?